tracer_notlinux.go

```diff
@@ -1,4 +1,4 @@
-// +build !linux
+// +build !linux linux,!nra_enabled
```

Comment:
I think this can just be `+build !nra_enabled`?

Also, rename the files to `tracer_nra_enabled` and ``tracer_nra_disabled`?